pkg/webhook/webhook_test.go

```diff
@@ -47,23 +47,21 @@ func TestGetBranchTagFromRef(t *testing.T) {
 func TestAzureDevopsWebhook(t *testing.T) {
 	const commit = "f00c3a181697bb3829a6462e931c7456bbed557b"
 	const repoURL = "https://dev.azure.com/fleet/git-test/_git/git-test"
-	gitjob := &v1.GitJob{
+	gitRepo := &v1alpha1.GitRepo{ // TODO update this to make tests pass
```

Comment:
Can this comment be removed?

Reply:
Oops, thanks for spotting this. Of course it could :o)